README.md

```diff
@@ -5,13 +5,6 @@ This group of tests was built to test microformats parsers. The individual tests
 The tests are broken into sets within the tests directory of this project. They are first grouped by version. Some parsers only support a single version of microformats. They are then subdivided by the type of microformat i.e. h-card.
 
 
-## NPM
-
-The test have been added to npm (Node Package Manager) and the latest version can be add to a project using the following command:
-
-    npm i microformat-tests --save
-
-
```

Comment:
What if, rather than removing this entirely, we provide a more useful installation example? Something like:

> To add the microformats test suite to your project using [npm](https://www.npmjs.com), run the following command, replacing `e9e2b90` with [a recent commit SHA](https://github.com/microformats/tests/commits/master):
>
> `npm install microformats/tests#e9e2b90 --save-dev `

Reply:
I'm not sure we _need_ to specify the commit. This complicates the installation instructions and the commit hash is saved into the lockfile (at least yarn does) so it will always resolve to the same commit unless re-installed. Running `yarn upgrade` automatically updates this. Also if you choose to not commit your lockfile, you're specifying to always use the latest `master`.

Example:
```
 "microformat-tests@https://github.com/aimee-gm/tests":
  version "0.1.26"
  resolved "https://github.com/aimee-gm/tests#3bd484edfab77a52c19922d1e312e6ce57339532"
```

So, we may be able to say:

> To add the latest microformats test suite to your project using npm, run the following command:
>
> `npm install microformats/tests --save-dev`
>
> Update to the latest microformats test suite with the following command:
>
> `npm update microformats/tests`

Reply:
That looks reasonable to me! Great addition to the documentation.